tests/end_to_end/http_server/large_files/src/index.ts

```diff
@@ -0,0 +1,92 @@
+import express, { Request } from 'express';
+import { access, readFile, stat } from 'fs/promises';
+import mime from 'mime';
+import { extname } from 'path';
+
+import { ls } from './list_files';
```

Comment:
For this whole serving index thing, it is cool, very cool, but can't we just use a simple Express middleware for this purpose? Like https://www.npmjs.com/package/serve-index?

Reply:
We're going to try and get this to work

Reply:
https://github.com/demergent-labs/azle/issues/1995